<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustInfoPrompt</class>
 <widget class="QDialog" name="CustInfoPrompt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Company Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address 1:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CompAddy">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="CompName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Remove">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QComboBox" name="InterestBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Very interested</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Some what Interested</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Not very Interested</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Never call again</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="ImportanceBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>104</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nice to Have</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Key</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Level of Interest</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Importance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address 2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CompAddy_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Edit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>115</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
